src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -533,4 +532,14 @@ public PresentationHint Hint
             set { hint = value; }
         }
     }
+
+    public class ResponseUnicodeCleaner
```

Comment:
> Cleaning unicode characters is not a responsibility of the ServiceControl class

It can be if communicating with ServiceControl requires it. The ServiceControl class is responsible for communicating, and should hide the details.

I don't like having a class with so little code in it too, but I'll admit that's a personal preference. Also currently the code isn't being reused so I prefer the method being inside the class using it.